code/modules/paperwork/paper.dm

```diff
@@ -29,6 +29,18 @@
 	var/list/stamped
 	var/rigged = 0
 	var/spam_flag = 0
+	var/contact_poison // Reagent ID to transfer on contact
```

Comment:
Just make the pen empty, have it cost 1 TC, and then the person can load with their own poison mix, then use a reagent datum. Less special casing is generally good.